content/sensu-go/5.5/reference/tessen.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Tessen"
+description: "Tessens sends anonymized data about the Sensu instance to Sensu Inc. You can use sensuctl to view and manage the Tessen configuration. Read the reference to configure Tessen."
+weight: 10
+version: "5.5"
+product: "Sensu Go"
+menu: 
+  sensu-go-5.5:
+    parent: reference
+---
+
+- [Configuring Tessen](#configuring-tessen)
+- [Specification](#tessen-specification)
+- [Example](#tessen-example)
+
+## What is Tessen?
+
+Tessen is the Sensu call-home service.
+Included with every Sensu backend, Tessen sends anonymized data about the Sensu instance to Sensu Inc. every six hours, including:
+
+- The Sensu version
+- The number of Sensu agents and backends in the cluster
+- Whether the Sensu instance uses an [enterprise license][4]
+
+All data submissions are logged for complete transparency and transmitted over HTTPS.
+[Read the blog post][1] to learn more about Tessen.
+
+## Configuring Tessen
+
+Tessen is enabled by default.
+You can use the [Tessen API][2] and [sensuctl][3] to view and manage the Tessen configuration.
+[Licensed][4] Sensu instances cannot opt out of Tessen.
+
+To see the status of Tessen:
+
+{{< highlight shell >}}
+tessen info
+{{< /highlight >}}
+
+To opt-out of Tessen:
+
+{{< highlight shell >}}
+tessen opt-out
+{{< /highlight >}}
+
+To opt-in to Tessen:
+
+{{< highlight shell >}}
+tessen opt-in
+{{< /highlight >}}
```

Comment:
Do you have examples for the output of these commands? Is there an available --skip-confirm flag?

Reply:
```
$ sensuctl tessen opt-in
Thank you so much for opting in!

$ sensuctl tessen opt-out
? Are you sure you would like to opt-out of tessen? We'd hate to see you go! Yes
I thought we were friends :( Remember, you can opt back in at any time!

$ sensuctl tessen opt-out --skip-confirm
I thought we were friends :( Remember, you can opt back in at any time!
```